pkg/driver/agent.go

```diff
@@ -110,14 +110,18 @@ func (ns *node) NodePublishVolume(
 
 	vol, mountInfo, err := GetVolAndMountInfo(req)
 	if err != nil {
-		goto PublishVolumeResponse
+		return nil, err
```

Comment:
should the error be wrapped like it is being done below.

Reply:
yes, we should, added.